src/prometheus_alert_rules/flapping.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: AlertRules
+  rules:
+  - alert: AlertRulesFlapping
+    expr: changes(ALERTS_FOR_STATE{severity = "warning"}[1h]) > 5
```

Comment:
TIL!
Apparently there also `ALERTS`, but the value there is "1", whereas for `ALERTS_FOR_STATE` it's timestamp.
How did you find out this exists?

Reply:
Through @err404r :)

Reply:
ALERTS is not working with changes for some reason, when I found this out I google and find out about ALERTS_FOR_STATE it is internal metrics which is used to track alert state in between prometheus restarts. According to official docs

Reply:
Would we need to replicate the alert per severity?

Reply:
BTW, should we move this alert to prometheus? Seems like a generic alert rule.
We probably don't want topology injected here.
(And also we don't want prometheus topology injected there.)